### ISSUE TITLE
Change NavigationDemoRegion to use Ryan's spriteHelpers for drawing

### ISSUE THREAD

**Issue by gyurgyma** [lavinrp/GameBackbone#99]
NavigationDemoRegion's drawing of sprites is all screwy. Update it using Ryan's spriteHelpers and my conversion class.

**Comment by lavinrp** [comment]
Blocked by #100

**Comment by lavinrp** [comment]
Should be good to go.

**Comment by gyurgyma** [comment]
NavigationDemoRegion.cpp: Remove the second include of NavigationTools.h
                                               Add spaces between "//" and comments 
                                               line 75 remove the "()" around "paths[1]" so its the same as line 70
                                               line 109 and maybe 117 use the typedefs that you defined.
NavigationDemoRegion.h: Add spaces between "//" and comments 
                                           line 68 make sure that your "spaces" are spaces and not tabs. this line is a tab
                                           line 76 use typedefs if they exist
                                           line 77 should we store this as a pointer or do you not see anyone inheriting?
NavigationTools.cpp: Remove GB namespaces before function calls
                                  line 34 if youre gunna make function param multiline, make them all separate lines

**Comment by gyurgyma** [comment]
Merged into Dev.
